.github/scripts/compile.sh

```diff
@@ -22,15 +22,41 @@
 # https://sipb.mit.edu/doc/safe-shell
 set -Eefuxo pipefail
 
-WORKING="$(pwd)"
 BUILD_FLAGS="$(rpmbuild --eval '%set_build_flags')"
 eval "${BUILD_FLAGS}"
 
-#region libwebsockets
-wget "${LWS_SOURCE_URL}" -O libwebsockets.tar.gz
-tar -zxf libwebsockets.tar.gz --one-top-level=lws-src --strip-components 1
 
-cmake -S "$WORKING/lws-src" -B "$WORKING/lws_build" \
+if [ -z "${REMOTE_SOURCES_DIR:-}" ]; then
+  # If no REMOTE_SOURCES_DIR present in env, we use $(pwd) as working dir
+  WORKING_DIR=$(pwd)
+  SKUPPER_DIR=$WORKING_DIR
```

Comment:
```suggestion
  WORKING_DIR="$(pwd)"
  SKUPPER_DIR="$WORKING_DIR"
```

this does not really matter, but IMO in bash scripts it's best to quote everything that has variables in it diligently

Reply:
I tried running https://www.shellcheck.net/, one thing that it likes to suggest is this quoting... and it's not wrong about it...